test/image/mocks/funnel_axis.json

```diff
@@ -0,0 +1,143 @@
+{
+    "data": [
+        {
+            "type": "funnel",
+            "orientation": "v",
+            "x": [
+                "A",
+                "B",
+                "C"
+            ],
+            "y": [
+                3,
+                2,
+                1
+            ],
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x",
+            "yaxis": "y"
+        },
+        {
+            "type": "funnel",
+            "orientation": "v",
+            "x": [
+                "A",
+                "B",
+                "C"
+            ],
+            "y": [
+                300,
+                200,
+                100
+            ],
+            "textposition": "inside",
+            "insidetextanchor": "start",
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x2",
+            "yaxis": "y2"
+        },
+        {
+            "type": "funnel",
+            "orientation": "h",
+            "x": [
+                30000,
+                20000,
+                10000
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "textposition": "inside",
+            "insidetextanchor": "end",
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x3",
+            "yaxis": "y3"
+        },
+        {
+            "type": "funnel",
+            "orientation": "h",
+            "x": [
+                3000000,
+                2000000,
+                1000000
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "textposition": "outside",
+            "cliponaxis": false,
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x4",
+            "yaxis": "y4"
+        }
+    ],
+    "layout": {
+        "width": 800,
+        "height": 800,
+        "dragmode": "pan",
+        "xaxis": {
+            "type": "category",
```

Comment:
You should need to set `type: 'category'` in these mocks. Please :hocho:

Reply:
Done in https://github.com/plotly/plotly.js/pull/3817/commits/c6155dbd1e2bd146848a0ada80cd7de44ada07ad.